frontend/package.json

```diff
@@ -24,8 +24,9 @@
     "@angular/youtube-player": "^12.1.3",
     "@auth0/angular-jwt": "^5.0.2",
     "@microsoft/signalr": "^5.0.8",
+    "@ng-bootstrap/ng-bootstrap": "^10.0.0",
     "@swimlane/ngx-charts": "^18.0.1",
-    "@watchdog-bsa/watchdog-js": "^0.3.8",
+    "@watchdog-bsa/watchdog-js": "^0.4.12",
```

Comment:
We should wait untill project api key checking endpoint will appear. Without it, the npm package cannot check for the existence of the api key and won't send any errors to collector

Reply:
All right, man ))